src/core/state/userChains/index.ts

```diff
@@ -52,10 +52,10 @@ const initialChains = () => {
 };
 
 const initialUserChainsOrder = () => {
-  const supportedMainnetChainIds = useBackendNetworksStore
+  const sortedSupportedChainIds = useBackendNetworksStore
     .getState()
-    .getSupportedMainnetChainIds();
-  return supportedMainnetChainIds;
+    .getSortedSupportedChainIds();
+  return sortedSupportedChainIds;
 };
 
 export const userChainsStore = createStore<UserChainsState>(
```

Comment:
We should be able to remove `migrations` here since `handleUpdateUserChains` is handling the diffing.